draft-ietf-rats-eat.md

```diff
@@ -417,23 +417,26 @@ Endorsement:
 
 # Top-Level Token Definition
 
-An EAT is a "message", a "token", or such whose content is a Claims-Set about an entity or some number of entities.
-An EAT MUST always contains a Claims-Set.
+An EAT MUST contain a claims set about an entity packaged as a CWT {{RFC8392}} or JWT {{RFC7519}}. This specification extends {{RFC8392}} and {{RFC7519}} by defining additional claims for attestation. All definitions, requirements, creation and validation procedures, security considerations, IANA registrations, etc. from CWT and JWT apply to the EAT format.
```

Comment:
I would like to retain my original two sentences here. I think they provide a crucial definition of an EAT that goes beyond CWT and JWT.  They define what can plug into the CDDL sockets mention below and what cannot.

I think it is important to have some text that says what an EAT is independent of it being a CWT/JWT.

Most of this paragraph is repetition of the opening paragraph in section 4. I'm not sure whether it is better here or in section 4, but it shouldn't be in both.

I think the text say what an EAT is independent of CWT/JWT should be separate of how EAT relates to a CWT/JWT.

Reply:
I agree when editing section by section it is difficult to control repetition. Will restore your two sentences though it is not clear to me how in the first two sentences of the change fail to get the point across (though "an entity" should have been "an entity or a set of entities".

Reply:
The first sentence of your changes says to me that and EAT MUST be a CWT or JWT implying that nothing else can be an EAT.

Reply:
I already put your two sentences back, so all good.